src/components/views/elements/EditableItemList.js

```diff
@@ -121,8 +121,8 @@ export default class EditableItemList extends React.Component {
         return (
             <form onSubmit={this._onItemAdded} autoComplete={false}
                   noValidate={true} className="mx_EditableItemList_newItem">
-                <Field id="newEmailAddress" label={this.props.placeholder}
-                       type="text" autoComplete="off" value={this.props.newItem}
+                <Field id="newAddress" label={this.props.placeholder}
```

Comment:
Maybe `newItem`...? Really these IDs should be `mx` scoped, so perhaps `mx_EditableItemList_newItemField` or something?

Reply:
Given that we can have multiple EditableLists rendered at once they should actually get some unique ID